public/app/plugins/datasource/tempo/traceql/traceql.ts

```diff
@@ -36,7 +49,18 @@ export const language = {
   tokenizer: {
     root: [
       // labels
-      [/[a-z_][\w.]*(?=\s*(=|!=|=~|!~))/, 'tag'],
+      [/[a-z_.][\w./_-]*(?=\s*(=|!=|>|<|>=|<=|=~|!~))/, 'tag'],
+
+      // all keywords have the same color
+      [
+        /[a-zA-Z_.]\w*/,
+        {
+          cases: {
+            '@keywords': 'type',
+            '@default': 'identifier',
+          },
+        },
+      ],
 
       // strings
       [/"([^"\\]|\\.)*$/, 'string.invalid'], // non-teminated string
```

Comment:
Just a minor typo that you can fix in your next commit. Should be `non-terminated`.